### PR TITLE
wildcard casing

### PR DESCRIPTION
**Pull request by trinity-1686a**
fix normalizer used for the `raw_lowercase` tokenanizer

related to #3918

### CODE REVIEW

File: quickwit/quickwit-query/src/tokenizers/tokenizer_manager.rs [hunk 1]
```diff
@@ -83,7 +89,7 @@ impl TokenizerManager {
             .get(tokenizer_name)
             .copied()?;
         let analyzer = if use_lowercaser {
-            LOWERCASE_TOKENIZER_NAME
+            RAW_LOWERCASE_TOKENIZER_NAME
```
Comment (trinity-1686a) [inline quickwit/quickwit-query/src/tokenizers/tokenizer_manager.rs:92]: the tokenizer here isn't really supposed to generate multiple tokens

File: quickwit/quickwit-search/src/root.rs [hunk 2]
```diff
@@ -1207,6 +1207,11 @@ pub async fn search_plan(
             .terms_grouped_by_field
             .values()
             .map(|terms: &HashMap<tantivy::Term, bool>| terms.len())
+            .sum::<usize>()
+        + warmup_info
+            .term_ranges_grouped_by_field
```
Comment (trinity-1686a) [inline quickwit/quickwit-search/src/root.rs:1212]: range queries (including wildcards) were not properly counted

**Comment by fulmicoton** [comment]
Can we have a unit test somewhere?

**Review by fulmicoton — CHANGES_REQUESTED** [review]
Please add a unit test. If it is difficult, an integration test will do.
Make sure it runs on elasticsearch if you go for the integration test.

**Review by fulmicoton — APPROVED** [review]

**Pull request merged**